api.Dockerfile

```diff
@@ -0,0 +1,42 @@
+# First step: build the assets
+FROM node:lts-alpine AS builder
+
+WORKDIR /
+ADD package.json yarn.lock ./
+ADD ./apis/core/package.json ./apis/core/
+ADD ./ui/package.json ./ui/
+
+# for every new package foo add
+#ADD ./packages/foo/package.json ./packages/foo/
+
+# install and build backend
+ENV NODE_ENV=
```

Comment:
Not sure what that means

Reply:
I take this from the previous Dockerfile in curation. @sandhose is that needed?

Reply:
This isn't set by the parent image, so no need to do that